src/pages/FetchedDetails.js

```diff
@@ -1,77 +1,83 @@
 import styled from 'styled-components/macro';
 import { NavLink } from 'react-router-dom';
+import Heading from '../components/Heading';
 
 export default function Detail({ savedIssue }) {
   return (
-    <Wrapper>
-      <Navlink to="/" aria-label="back">
-        <svg
-          width="38"
-          height="38"
-          fill="var(--bg-color-primary)"
-          viewBox="0 0 16 16"
-        >
-          <title>Back</title>
-          <path d="M8 0a8 8 0 1 0 0 16A8 8 0 0 0 8 0zm3.5 7.5a.5.5 0 0 1 0 1H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5z" />
-        </svg>
-      </Navlink>
-      <dl>
+    <>
+      <header>
+        <Heading title="DETAIL" />
+      </header>
+      <Wrapper>
+        <Navlink to="/" aria-label="back">
+          <svg
+            role="img"
+            aria-label="Back arrow"
+            width="38"
+            height="38"
+            fill="var(--bg-color-primary)"
+            viewBox="0 0 16 16"
+          >
+            <title>Back</title>
+            <path d="M8 0a8 8 0 1 0 0 16A8 8 0 0 0 8 0zm3.5 7.5a.5.5 0 0 1 0 1H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5z" />
+          </svg>
+        </Navlink>
         <FlexContainer>
-          <div>
+          <DL>
             <DT>Number:</DT>
             <DD>{savedIssue.number}</DD>
             <DT>Issue ID:</DT>
             <DD>{savedIssue.id}</DD>
             <DT>User:</DT>
             <DD>{savedIssue.user.login}</DD>
-          </div>
-          <div>
-            <DD>
-              <Avatar
-                src={savedIssue.user.avatar_url}
-                alt="avatar"
-                width="100"
-                height="100"
-              />
-            </DD>
-          </div>
+          </DL>
+          <Avatar
+            src={savedIssue.user.avatar_url}
+            alt="avatar"
```

Comment:
I think it would be nicer to either leave the `alt` empty or add the name of the user.